app/src/main/java/crux/bphc/cms/fragments/MyCoursesFragment.java

```diff
@@ -524,6 +552,17 @@ public void markAllAsRead(int position){
                 unreadCount.setVisibility(count == 0 ? View.INVISIBLE : View.VISIBLE);
                 Toast.makeText(getActivity(), "Marked all as read", Toast.LENGTH_SHORT).show();
             }
+
+            public void setFavoriteStatus(int position, boolean isFavourite){
```

Comment:
Insert whitespace between closing parenthesis of parameter list and opening braces.